src/pages/HomePage.tsx

```diff
@@ -94,67 +97,39 @@ export const HomePage = () => {
                 </Stack>
               </Grid>
               <Grid item xs={6}>
-                <Stack direction="column" alignItems="center">
+                <Stack direction="column">
                   <Stack direction="row" width="100%" spacing={2}>
-                    <Box
-                      height={111}
-                      width={111}
-                      borderRadius="50px 50px 27px 50px"
-                      boxShadow="hsl(0deg 0% 0% / 0.2) -20px 20px"
-                    >
-                      <HomepagePPIcon />
-                    </Box>
-                    <Stack direction="column" justifyContent="flex-end">
-                      <Typography variant="h4" fontWeight="lighter">
-                        1,483,238
-                      </Typography>
-                      <Typography variant="h6" fontWeight="lighter">
-                        pp earned
-                      </Typography>
-                    </Stack>
+                    <HomepageStatDisplay
+                      title="pp earned"
+                      value={totalPPEarned}
+                      icon={<HomepagePPIcon />}
+                      lessRoundedCorner="bottom-right"
+                      shadowDirection="bottom-left"
+                    />
                   </Stack>
                   <Stack
-                    direction="row"
+                    direction="row-reverse"
                     width="100%"
                     mt={-1}
                     spacing={2}
-                    justifyContent="flex-end"
-                    alignItems="flex-end"
+                    justifyContent="flex-start"
                   >
-                    <Stack direction="column">
-                      <Typography variant="h4" fontWeight="lighter">
-                        92,383,238
-                      </Typography>
-                      <Typography variant="h6" fontWeight="lighter">
-                        scores set
-                      </Typography>
-                    </Stack>
-                    <Box
-                      height={111}
-                      width={111}
-                      borderRadius="50px 50px 50px 27px"
-                      boxShadow="hsl(0deg 0% 0% / 0.2) 20px 20px"
-                    >
-                      <HomepageScoresIcon />
-                    </Box>
+                    <HomepageStatDisplay
+                      title="scores set"
+                      value={totalScoresSet}
+                      icon={<HomepageScoresIcon />}
+                      lessRoundedCorner="bottom-left"
+                      shadowDirection="bottom-left"
```

Comment:
i think?

```suggestion
                      shadowDirection="bottom-right"
```

![image](https://i.cmyui.xyz/O86Ulkd9GNnYeJw.png)